macroeco_distributions/macroeco_distributions.py

```diff
@@ -657,18 +661,18 @@ def zipf_solver(ab):
     ab = check_for_support(ab, lower = 1)
     par0 = 1 + len(ab) / (sum(np.log(2 * np.array(ab))))
     def zipf_func(x):
-        return -zipf_ll(ab, x)
-    par = optimize.fmin(zipf_func, x0 = par0)[0]
+        return -zipf_ll(ab, exp(x))
```

Comment:
Oh jeez.  Good catch. I branched off of my other pull request (#33) and didn't even realize it because of all the whitespace changes.  I'll kill this PR and submit a proper one.